src/zig.rs

```diff
@@ -446,20 +446,10 @@ impl Zig {
                         format!("--sysroot={}", libc.join("mingw").display()),
                     );
                 } else if raw_target.contains("apple-darwin") {
-                    if let Some(sdkroot) = Self::macos_sdk_root() {
-                        cmd.env(
-                            bindgen_env,
-                            format!(
-                                "-I{} -F{} -DTARGET_OS_IPHONE=0",
-                                sdkroot.join("usr").join("include").display(),
-                                sdkroot
-                                    .join("System")
-                                    .join("Library")
-                                    .join("Frameworks")
-                                    .display()
-                            ),
-                        );
-                    }
+                    cmd.env(
```

Comment:
I'd prefer to use this when `SDKROOT` isn't specified because some C/C++ dependencies might actually require more system libraries, zig only bundles `libSystem.tbd`.

Reply:
Understood. However, I itched by `-DTARGET_OS_IPHONE=0` part. Should be ignore this ad-hoc solution?

Reply:
Ah, it also used by `execute_compiler`. NM. I'm okay with it. Thanks for your clarification!